<?xml version="1.0"?>
<launch>
<arg name="port" default="/dev/ttyUSB0"/>
<arg name="baud" default="4800"/>
<param name="port" value="$(arg port)"/>
<param name="baud" value="$(arg baud)"/> 

    <node pkg="gps_driver" type="driver.py" name="gps_talker" output="screen">
               
        
    </node>

</launch>